packages/consensus/src/types/chain_state.cairo

```diff
@@ -64,16 +64,17 @@ pub impl BlockValidatorImpl of BlockValidator {
         let block_height = self.block_height + 1;
 
         let prev_block_time = *self.prev_timestamps[self.prev_timestamps.len() - 1];
+
+        let median_time_past = compute_median_time_past(self.prev_timestamps);
```

Comment:
Can you please add a comment here that it is a MTP of the previous block? To emphasize this